main.tf

```diff
@@ -1,3 +1,20 @@
+locals {
+  enabled = module.this.enabled
+  # We do not use coalesce() here because it is OK if local.bucket_name is empty.
+  bucket_name = var.bucket_name == null || var.bucket_name == "" ? module.bucket_name.id : var.bucket_name
+}
+
+module "bucket_name" {
```

Comment:
Due to a new module addition, please update the docs via `make docs` or `make readme/docs`

Reply:
Fix with https://github.com/benjamin-hg/terraform-aws-lb-s3-bucket/commit/4082294d2a5070cd747385efd2ff8159ad8f065c